.gitignore

```diff
@@ -41,6 +41,7 @@ internet_identity.did
 codes.txt
 
 backend.wasm.gz
+ic-btc-canister.wasm.gz
 backend-v*.wasm.gz
```

Comment:
At some time someone will replace this list with `*.wasm.gz` :laughing:

Reply:
I hope 😂